scripts/core/program.yml

```diff
@@ -10,6 +10,67 @@ type: header
 desc: "Intel $OneApi Level-Zero Runtime APIs for Program"
 ordinal: "2"
 --- #--------------------------------------------------------------------------
+type: enum
+desc: "Program metadata property type."
+name: $x_program_metadata_type_t
+etors:
+    - name: UNKNOWN  # TODO: Is this necessary?
```

Comment:
this enum is to define the type, so UR knows what to do with it, if it not known, kind of contradicts the purpose, no?

Reply:
Unknown is actually in [PI already](https://github.com/intel/llvm/blob/sycl/sycl/include/sycl/detail/pi.h#L709-L715) which I was unsure about.

Reply:
I have removed `UNKNOWN` for now. If its actually being used in SYCL RT we can add it back in later.